docs/literate/src/files/scalar_linear_advection_1d.jl

```diff
@@ -377,12 +381,13 @@ semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition_sine_wave
 
 # Again, combining all definitions and the function that calculates the right-hand side, we define the ODE and
 # solve it until `t=2` with OrdinaryDiffEq's `solve` function and the Runge-Kutta method `RDPK3SpFSAL49()`.
+tspan = (0.0, 2.0)
 ode_trixi  = semidiscretize(semi, tspan)
 
 sol_trixi = solve(ode_trixi, RDPK3SpFSAL49(), abstol=1.0e-6, reltol=1.0e-6, save_everystep=false);
 
 # We add a plot of the new approximated solution to the one calculated before.
-plot!(vec(x), vec(sol_trixi.u[end]), label="solution at t=$(tspan[2]) with Trixi.jl", legend=:topleft, linestyle=:dash, lw=2)
+plot!(sol_trixi, label="solution at t=$(tspan[2]) with Trixi.jl", legend=:topleft, linestyle=:dash, lw=2)
```

Comment:
@ranocha With Trixi's plot function the label is not shown in the picture. Is there any simple solution for this? I couldn't find any.

Reply:
What code did you run to *not* get the label? I get

![tmp](https://user-images.githubusercontent.com/12693098/139865913-8fa7d803-7a5d-4139-996d-b8b8877057f1.png)

from the line above and the code setting up the required data structures...

Reply:
Never mind, it's working for me as well now.